src/condition.cpp

```diff
@@ -1373,6 +1376,62 @@ void conditional_t::set_query( const JsonObject &jo, std::string_view member, bo
     };
 }
 
+void conditional_t::set_query_tile( const JsonObject &jo, std::string_view member, bool is_npc )
+{
+    std::string type = jo.get_string( member.data() );
+    var_info target_var = read_var_info( jo.get_object( "target_var" ) );
+    std::string message;
+    if( jo.has_member( "message" ) ) {
+        message = jo.get_string( "message" );
+    }
+    dbl_or_var range;
+    if( jo.has_member( "range" ) ) {
+        range = get_dbl_or_var( jo, "range" );
+    }
+    condition = [type, target_var, message, range, is_npc]( dialogue & d ) {
+        std::optional<tripoint> loc;
+        Character *ch = d.actor( is_npc )->get_character();
+        if( ch && ch->as_avatar() ) {
+            avatar *you = ch->as_avatar();
+            if( type == "anywhere" ) {
+                if( !message.empty() ) {
+                    static_popup popup;
+                    popup.on_top( true );
+                    popup.message( "%s", message );
+                }
+                tripoint center = d.actor( is_npc )->pos();
+                const look_around_params looka_params = { true, center, center, false, true, true, false };
```

Comment:
I think the last boolean here refers to `change_lv` right? Would it be possible to add another field like `range` that only exists for the "anywhere" mode this time and lets you toggle this to be on or off in the eoc itself? (defaulting to `false` like here)